fulllogging.properties

```diff
@@ -0,0 +1,17 @@
+# Global logging properties.
```

Comment:
We probably don't want this. JmDNS logging can, and has, revealed network funkiness that we've needed to assist users with in the past. It would probably be better to have a JUL->SLF4J->LOG4J chain instead of two different loggers running at once. See http://www.slf4j.org/legacy.html